content/applications/finance/expenses.rst

```diff
@@ -336,49 +408,71 @@ there are four options:
   reporting and configuration menus in the *Expenses* app.
 
 Users who are able to approve expense reports (typically managers) can easily view all expense
-reports to validate. Go to :menuselection:`Expenses app --> Expense Reports  --> Reports to
-Approve`. This view lists all the expense reports that have been submitted but not approved, as
-noted by the :guilabel:`Submitted` tag in the status column.
+reports they have access rights to. Go to :menuselection:`Expenses app --> Expense Reports`, and a
+list appears with all expense reports that have a status of either :guilabel:`To Submit`,
+:guilabel:`Submitted`, :guilabel:`Approved`, :guilabel:`Posted`, or :guilabel:`Done`. Expense
+reports with a status of :guilabel:`Refused` are hidden in the default view.
 
-.. image:: expenses/reports-to-approve.png
+.. image:: expenses/expense-reports-list.png
    :align: center
    :alt: Reports to validate are found on the Reports to Approve page.
 
+When viewing expense reports, there is a panel of filters to chose from on the left side. The three
+categories that filters can be applied on are :guilabel:`Status`, :guilabel:`Employee`, and
+:guilabel:`Company`. To view only expense reports with a particular status, click the checkbox next
+to the status to display the expense reports with that status. Click the checkbox again to hide the
+reports with that status. To view expense reports for a particular employee and/or company, click on
+the employee name and/or company name in the :guilabel:`Employee` and :guilabel:`Company` sections.
+
 Reports can be approved in two ways (individually or several at once) and refused only one way. To
 approve multiple expense reports at once, remain in the list view. First, select the reports to
 approve by clicking the check box next to each report, or click the box next to :guilabel:`Employee`
-to select all reports in the list. Next, click on the :guilabel:`⚙️ Action (gear)` icon, then click
-:guilabel:`Approve Report`.
+to select all reports in the list.
+
+.. important::
+   Only reports with a status of :guilabel:`Submitted` can be approved. It is recommended to only
+   display the submitted reports by adjusting the status filter on the left side by only having the
+   :guilabel:`Submitted` filter checked off.
```

Comment:
The idiom "checked off" I don't think works in this doc. It's usually in reference to to-do lists and productivity, as in, something was "done," not so much "enabled." As well idioms have a tendency to translate poorly into other languages so I'd be be more specific throughout the doc on every instance it shows up, e.g.:
```suggestion
   :guilabel:`Submitted` filter enabled.
```

Reply:
That makes a lot of sense- I changed all the check boxes where it applies to selecting filters/enabling items. I left the other "check" works when it was explaining how to select a few reports vs selecting the entire list (you'd click a different box for that). I *think* that makes sense- but if you'd like those parts changed as well, LMK.

Basically, "checked off" is gone, but "check the box" I kept.